oshc/main/models.py

```diff
@@ -2,5 +2,28 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.forms import ModelForm, Textarea
 
 # Create your models here.
+
+
+class chatSession(models.Model):
+    title = models.CharField(max_length=128, help_text="Require title of the session")
+    profile_name = models.CharField(max_length=128, help_text="Require name of the person")
+    profile_url = models.URLField(help_text="Require url of the person")
+    description = models.CharField(max_length=512, help_text="Require description of the session")
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
+    register_url = models.URLField(help_text="Require url for the registration of event")
+
+    def __str__(self):
+        return self.title
+
+
+class PostModelForm(ModelForm):
+    class Meta:
+        model = chatSession
+        fields = ('title', 'profile_name', 'profile_url', 'description', 'start_date', 'end_date', 'register_url')
```

Comment:
All the model fields will be displayed on the admin page by default. You don't need to add them explicitly.